tools/accuracy_checker/accuracy_checker/launcher/dlsdk_launcher.py

```diff
@@ -553,60 +572,34 @@ def set_nireq():
                 self._num_requests = 1
             else:
                 self._num_requests = self.auto_num_requests()
-
-        if hasattr(self, 'plugin'):
-            del self.plugin
         if log:
             print_info('IE version: {}'.format(ie.get_version()))
         if self._is_multi():
-            self._create_multi_device_plugin(log)
+            self._prepare_multi_device(log)
         else:
-            self.plugin = ie.IEPlugin(self._device)
             self.async_mode = self.get_value_from_config('async_mode')
             set_nireq()
 
             if log:
-                print_info('Loaded {} plugin version: {}'.format(self.plugin.device, self.plugin.version))
-        if self._device_ids:
-            correct_id = [device_id for device_id in self._device_ids if device_id is not None]
-            if correct_id:
-                self.plugin.set_config({'DEVICE_ID': correct_id[0]})
+                self._log_versions()
+
         cpu_extensions = self.config.get('cpu_extensions')
         if cpu_extensions and 'CPU' in self._devices_list():
             selection_mode = self.config.get('_cpu_extensions_mode')
             cpu_extensions = DLSDKLauncher.get_cpu_extension(cpu_extensions, selection_mode)
-            self.plugin.add_cpu_extension(str(cpu_extensions))
+            self.ie_core.add_extension(str(cpu_extensions), 'CPU')
         gpu_extensions = self.config.get('gpu_extensions')
         if gpu_extensions and 'GPU' in self._devices_list():
-            self.plugin.set_config('CONFIG_FILE', str(gpu_extensions))
+            self.ie_core.add_extension(str(gpu_extensions), 'GPU')
```

Comment:
I suppose for GPU we still need to `StConfig` since `AddExtension` does not work for GPU.

Reply:
Thanks, fixed